client/src/components/PersonalInformation.tsx

```
@@ -0,0 +1,329 @@
+            </Grid>
+
+            {/* Weight */}
+            <Grid item xs={12} md={6}>
+              <TextField
+                fullWidth
+                label="Weight (kg)"
+                id="weight"
+                name="weight"
+                type="number"
+                placeholder="Enter weight"
+                required></TextField>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)